Makefile

```diff
@@ -157,8 +157,8 @@ mypy:
 
 .PHONY: format
 format:
-	isort .
-	black .
+	isort src tests
+	black src tests
```

Comment:
Is this change important? Are there any files that we don't want to format?

Reply:
venv